templates/patterns/button/button.ui_patterns.yml

```diff
@@ -2,9 +2,6 @@ button:
   label: "Button"
   description: "Buttons indicate actions the users can take."
   variants:
-    default:
-      label: "Default"
-      description: "The default button is used for important links on the page."
     primary:
```

Comment:
We cannot remove variants etc. since they end up in configuration here and there. UI patterns are essentially our API, the exercise is really trying to fit the new ECL component into the "old" pattern, without changing anything of the pattern definition. In case we will need to change it then we might need an upgrade path.

Reply:
ok :)